README.md

```diff
@@ -30,6 +30,18 @@ To see Flask app: navigate to http://localhost:5000
 
 To see DNS admin panel: navigate to https://localhost:10000/ and log in with `root`/`highentropy`
 
+To run scapy from victim or attacker, install git:
+```bash 
```

Comment:
The attacker already has scapy installed: https://github.com/CompSec2/Coursework/blob/master/attacker/requirements.txt & https://github.com/CompSec2/Coursework/blob/master/attacker/Dockerfile#L4

It would be easier to just add the requirement for scapy to be installed to the requirements.txt of the victim as well: https://github.com/CompSec2/Coursework/blob/master/victim/requirements.txt

Reply:
This would mean you always have it available, so no need to install it after every rebuild of the container